packages/thirdweb/src/utils/any-evm/zksync/isZkSyncChain.ts

```diff
@@ -1,12 +1,24 @@
 import type { Chain } from "../../../chains/types.js";
+import { withCache } from "../../promise/withCache.js";
 
-export function isZkSyncChain(chain: Chain) {
-  return (
-    chain.id === 324 ||
-    chain.id === 300 ||
-    chain.id === 302 ||
-    chain.id === 11124 ||
-    chain.id === 282 || // cronos zkevm testnet
-    chain.id === 388 // cronos zkevm mainnet
+export async function isZkSyncChain(chain: Chain) {
+  const stack = await getChainStack(chain.id);
```

Comment:
In here, add a .catch() that defaults to checking the old hardcoded list behavior, that way we have a fallback if things go wrong